util/AeroDA/calc_increment_ens.fd/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
-  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fp-model strict -convert big_endian")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -g -traceback -fp-model strict -convert big_endian")
```

Comment:
The `util` can be done project wide rather than utility by utility.
I am surprised this is not being handled by https://github.com/MichaelLueken-NOAA/GSI/blob/feature/add_traceback/util/cmake/gsiutils_compiler_flags.cmake that is being included at https://github.com/MichaelLueken-NOAA/GSI/blob/98a45b09c41e7df2eb8f4996499d4b4e99b8e22e/util/CMakeLists.txt#L118

I can take a look.